### DISCUSSION TITLE
Danfoss 014g2420 not pairing

### DISCUSSION THREAD

**Discussion by Speedy1496** [Koenkk/zigbee2mqtt#16957] [Devices]
Hello,
I just bought a 014G2420 TRV Zigbee. (Which is the only one we can find now...)
I'm trying to pair but when I do teh process : 
- The TRV is found by Z2M
- It is added as an unssuported device
- Failed ot interview the device
- and I have the message the device has left the network.

Does someone had paired this head to Z2M? 
I found some discussion last year and it's were supposed to work

Thank you very much

Z2M 1.30.2-1
Sonoff Zigbee Dongle-E

Sébastien

**Comment by Speedy1496** [comment]
2023-03-07 22:22:14Device '0x04cd15fffe7a5c17' joined
Info 2023-03-07 22:22:14MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x04cd15fffe7a5c17","ieee_address":"0x04cd15fffe7a5c17"},"type":"device_joined"}'
Info 2023-03-07 22:22:14Starting interview of '0x04cd15fffe7a5c17'
Info 2023-03-07 22:22:15MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x04cd15fffe7a5c17","ieee_address":"0x04cd15fffe7a5c17","status":"started"},"type":"device_interview"}'
Error 2023-03-07 22:22:35Failed to interview '0x04cd15fffe7a5c17', device has not successfully been paired
Info 2023-03-07 22:22:35MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x04cd15fffe7a5c17","ieee_address":"0x04cd15fffe7a5c17","status":"failed"},"type":"device_interview"}'
Warning 2023-03-07 22:22:44Device '0x04cd15fffe7a5c17' left the network
Info 2023-03-07 22:22:44MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x04cd15fffe7a5c17","ieee_address":"0x04cd15fffe7a5c17"},"type":"device_leave"}'

**Comment by Speedy1496** [comment]
This morning the device pairing worked after 10 tries.
I don't know why it work now.

Thank you

**Comment by Speedy1496** [comment]
For information the 014G2420 Danfoss Smart valve is regognized as 014G2461 device in Z2M

**Reply by lux73** [reply]
that's correct - as i remember correctly 014G2420 vs 014G2461 differs only for the attached Valve Adapters

**Reply by flowg22** [reply]
Hello,
maybe i have the same problem. i got just the 014G2460 instead of the ....61 or ..63, which are listed at https://www.zigbee2mqtt.io/supported-devices/#v=Danfoss
i also use an Sonoff Zigbee Dongle

is there any different between the ....60, ...61 or 63... instead of the valve adapters?

thank you

**Reply by flowg22** [reply]
I was wrong. The only problem was that I didn't have the latest version of Z2M. Update solved everything.

**Comment by Speedy1496** [comment]
I think I found what happened.
I tried to pair with a Sonoff Dongle-E (EFR32MG21) it was not working. I have updated my homeassistant and the route has been redirected on an other device (a Lixee).
I think it has been properly pair when the route were going through an other device than the EFR32MG21

**Comment by Schlicki** [comment]
I have the same problem with a [https://www.zigbee2mqtt.io/devices/701721.html](url) device, which is the same with other label.

Joins network and loses connection after 20 seconds while pairing with a Sonoff dongle.
